src/AutoHideTab.cpp

```diff
@@ -537,6 +542,37 @@ void CAutoHideTab::mouseMoveEvent(QMouseEvent* ev)
     Super::mouseMoveEvent(ev);
 }
 
+//============================================================================
+void CAutoHideTab::dragEnterEvent(QDragEnterEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)) {
+        if (!d->TimeSinceDragOver.isValid()) {
+            d->TimeSinceDragOver.restart();
+            ev->accept();
+        }
+        else if (d->TimeSinceDragOver.hasExpired(500)) {
+            ev->accept();
+        }
+    }
+}
+
+//============================================================================
+void CAutoHideTab::dragLeaveEvent(QDragLeaveEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)) {
+        d->TimeSinceDragOver.invalidate();
+        d->forwardEventToDockContainer(ev);
+    }
+}
+
+//============================================================================
+void CAutoHideTab::dragMoveEvent(QDragMoveEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)
+        && d->TimeSinceDragOver.isValid()
+        && d->TimeSinceDragOver.hasExpired(500)) {
+        d->TimeSinceDragOver.invalidate();
+        d->DockWidget->autoHideDockContainer()->collapseView(false);
```

Comment:
If there are multiple auto hide tabs, then moving the mouse over the tabs will show multiple auto hide dock widgets because if a new one is shown, the old one is not closed,